From 82e5a7197e043690c6c41749411881f09e4b179f Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Sat, 11 Mar 2023 13:02:38 -0300
Subject: [PATCH 2/2] db: don't warn the user when db is not found

Expected code path when db needs to be initialized (first run).

Upstream-Status: Inappropriate [lmp specific]

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 src/lib/db.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib/db.c b/src/lib/db.c
index 8532904..23af9ff 100644
--- a/src/lib/db.c
+++ b/src/lib/db.c
@@ -1444,7 +1444,7 @@ static CK_RV db_get_version(sqlite3 *db, unsigned *version) {
     sqlite3_stmt *stmt;
     int rc = sqlite3_prepare_v2(db, sql, -1, &stmt, NULL);
     if (rc != SQLITE_OK) {
-        LOGW("Cannot prepare version query: %s\n", sqlite3_errmsg(global.db));
+        LOGV("Cannot prepare version query: %s\n", sqlite3_errmsg(global.db));
         *version = DB_EMPTY;
         return CKR_OK;
     }
-- 
2.34.1

